eventtracking/backends/segment.py

```diff
@@ -0,0 +1,64 @@
+"""Event tracking backend that sends events to segment.com"""
+
+from __future__ import absolute_import
+
+try:
+    import analytics
+except ImportError:
+    analytics = None
+
+
+class SegmentBackend(object):
+    """
+
+    Send events to segment.com
+
+    It is assumed that other code elsewhere initializes the segment.com API and makes calls to analytics.identify.
```

Comment:
Do we know if/that this will happen in time?  That is, before this backend would get called?